language/and_spec.rb

```diff
@@ -11,6 +11,12 @@
   it "evaluates to the first condition not to be true" do
     ("yes" && 1 && nil && true).should == nil
     ("yes" && 1 && false && true).should == false
+    value = nil
+    (value && nil).should == nil
+    (value && false).should == nil
+    value = false
+    (value && nil).should == false
+    (value && false).should == false
```

Comment:
Just a style issue:
Could you put the new examples above since they are simpler and with an empty line between the 2-operands and the 4-operands examples?